.travis.yml

```diff
@@ -1,9 +1,22 @@
 language: python
 python:
-  - 2.7
```

Comment:
This python is used to run tox, not django. Tox create virtualenv to run django according tox.ini which contains python 2.7 and others.